### PR TITLE
docs: remove joke from pronounce

### PR DESCRIPTION
**Pull request by dundargoc**
N-Jim does not sounds like "Ninja", and the joke doesn't really land.

### CODE REVIEW

**Comment by clason** [comment]
Here's the upstream version, of reference:

```
                                                        *pronounce*
Vim is pronounced as one word, like Jim, not vi-ai-em.  It's written with a
capital, since it's a name, again like Jim.
```

Maybe keep the section, but remove the jokey bits?

**Comment by dundargoc** [comment]
I initially wrote that in the commit message that "this is cringe" but I couldn't come up with a "professional" way to say it.

File: runtime/doc/intro.txt [hunk 1]
```diff
@@ -22,8 +22,7 @@ is not located in the default place.  You can jump to subjects like with tags:
 Use CTRL-] to jump to a subject under the cursor, use CTRL-T to jump back.
 
 							*pronounce*
-Vim is pronounced as one word, like Jim.  So Nvim is N-Jim, which sounds like
-"Ninja".  Starting Nvim is like performing a roundhouse kick.
+Vim is pronounced as one word, like Jim.  So Nvim is N-Jim.
```
Comment (clason) [inline runtime/doc/intro.txt:25]:
```suggestion
Vim is pronounced as one word, like Jim.  So Nvim is N-Vim, two syllables.
```

**Comment by llllvvuu** [comment]
On the topic of personality, there are a number of flairs like "brain-damaged" and "BASIC BITCH" in the code, which I personally have no problem with, but who knows, it's a critical world out there

**Comment by dundargoc** [comment]
> On the topic of personality, there are a number of flairs like "brain-damaged" and "BASIC BITCH" in the code, which I personally have no problem with (I'm a fan of Linux), but who knows, it's a critical world out there

The difference is that those are internal and not user-facing comments. They're also not lethally cringe.

File: runtime/doc/intro.txt [hunk 2]
```diff
@@ -22,8 +22,7 @@ is not located in the default place.  You can jump to subjects like with tags:
 Use CTRL-] to jump to a subject under the cursor, use CTRL-T to jump back.
 
 							*pronounce*
-Vim is pronounced as one word, like Jim.  So Nvim is N-Jim, which sounds like
-"Ninja".  Starting Nvim is like performing a roundhouse kick.
+Vim is pronounced as one word, like Jim.  So Nvim is N-Vim, two syllables.
```
Comment (clason) [inline runtime/doc/intro.txt:25]:
```suggestion
Vim is pronounced as one word, like Jim.  So Nvim is "En-Vim", two syllables.
```

**Review by clason — APPROVED** [review]

**Pull request merged**